content/en/tracing/trace_collection/custom_instrumentation/ios/otel.md

```diff
@@ -0,0 +1,327 @@
+---
+title: iOS and tvOS Custom Instrumentation using OpenTelemetry API
```

Comment:
We recently published a page for this content: https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/ios?tab=cocoapods.

I'm not sure which is most relevant, but we should reconcile these.

Reply:
rebase done